acro/acro.py

```diff
@@ -143,6 +143,20 @@ def print_outputs(self) -> None:
                 print(f"{key}: {item}")
             print("\n")
 
+    def custom_output(self, filename: str) -> None:
+        """Adds an unsupported output to the results dictionary.
+
+        Parameters
+        ----------
+        filename : str
+            The name of the file that will be added to the list of the outputs.
+        """
+        self.__add_output(
+            command=None, summary=None, outcome=None, output=os.path.abspath(filename)
```

Comment:
This causes the following to be written to the results json:
```
    "output_9_2023-04-16-10393905": {
        "command": null,
        "summary": null,
        "outcome": null,
        ...
  }
```

Instead of setting these to None, I think command should be set to "custom", summary should be set to "review" and outcome can be set to "{}" (which is similar to the regression outcomes).